MoreLinq/MoreLinq.csproj

```diff
@@ -133,8 +133,15 @@
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
     <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
```

Comment:
Is this needed?

Reply:
These are actually optional but:
- the first one : Include PDB in the built .nupkg
- the second one : Declare that the Repository URL can be published to NuSpec
- the last one : Embed source files that are not tracked by the source control manager to the PDB (only useful if you generate files during the build)

as I didn't really know what to add or not I prefered to use this way. What do you think?

Reply:
no problem !